src/constants/MarkdownFiles/posts/2025-07-13-gsoc-25-MebinThattil-week06.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "GSoC ’25 Week 06 Update by Mebin J Thattil"
+excerpt: "Optimizations and reading documentation"
+category: "DEVELOPER NEWS"
+date: "2025-07-13"
+slug: "2025-07-13-gsoc-25-mebinthattil-week6"
+author: "@/constants/MarkdownFiles/authors/mebin-thattil.md"
+tags: "gsoc25,sugarlabs,week06,mebinthattil,speak_activity"
+image: "assets/Images/GSOCxSpeak.png"
+---
+
+<!-- markdownlint-disable -->
+
+# Week 06 Progress Report by Mebin J Thattil
+
+**Project:** [Speak Activity](https://github.com/sugarlabs/speak)  
+**Mentors:** [Chihurumnaya Ibiam](https://github.com/chimosky), [Kshitij Shah](https://github.com/kshitijdshah99)  
+**Assisting Mentors:** [Walter Bender](https://github.com/walterbender), [Devin Ulibarri](https://github.com/pikurasa)  
+**Reporting Period:** 2025-07-07 - 2025-07-13
+
+---
+
+## Goals for This Week
+
+- **Goal 1:** Improve dataset quality  
+- **Goal 2:** Fine-tune the model again and evaluate performance  
+- **Goal 3:** Work on mechanics for voice switching and personas inside Sugar  
+
+---
+
+## This Week’s Progress
+
+### **1. Improving the dataset**
+
+- The fine-tuning done using the previous dataset wasn’t up to expectations.  
```

Comment:
This section that you have in bullets seems more like prose to me. Perhaps just write it as a paragraph?

Also, it would be nice to have some more insight into _why_ it doesn't meet expectations. For example, what were the flags that this wasn't working? Or perhaps you could link to a page of your results data.